libs/barista-components/form-field/src/form-field-control.ts

```diff
@@ -40,7 +40,7 @@ export abstract class DtFormFieldControl<T> {
   readonly id: string;
 
   /** Gets the NgControl for this control. */
-  readonly ngControl: NgControl | null;
+  readonly ngControl?: NgControl | null;
```

Comment:
This would be a breaking change or? If somebody is relying on this in his script he would get currently always a value now he can get undefined as well...

Reply:
this is the least breaking i could make it. otherwise i would break all new form-field compatible components like switch/checkbox/...